css/design_system/utils/spacing.scss

```diff
@@ -0,0 +1,66 @@
+$space-types:
+  "p" "padding",
+  "m" "margin";
+
+$widths:
+  "0" "0",
+  "1" "4",
+  "2" "8",
+  "3" "16",
+  "4" "24",
+  "5" "40";
```

Comment:
Should we be taking these from the tokens/spacing file?

Reply:
Not sure what you mean. Mind adding more context? These are the intervals from that file, I followed a similar convention to how borders are generated.

Reply:
Yea border may not be done in the best way as a lot of things were being set up along with that PR. I'm thinking this is where we should be utilizing the tokens that we defined in Savile:
```
$widths:
  "0" "--s-space-0",
  "1" "--s-space-4",
  ...
```

Otherwise doesn't seem like we are really utilizing these spacing variables.

Reply:
Got it. Change coming...

Reply:
Updated in 6938fda